Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM python:3-alpine
 
-ARG KUBE_VERSION=1.13.7
+ARG KUBE_VERSION=1.19.6
 ENV HOME=/srv
 WORKDIR /srv
 
-RUN apk add --no-cache curl ca-certificates && \
-    pip --no-cache-dir --disable-pip-version-check --quiet install awscli
-RUN curl -f -s -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBE_VERSION}/bin/linux/amd64/kubectl && \
-    chmod +x /usr/local/bin/kubectl && \
-    kubectl version --client
+RUN apk add --no-cache curl ca-certificates \
+&& pip --no-cache-dir --disable-pip-version-check --quiet install awscli \
```

Comment:
I'd say that this changes are not necessary and they are out of the context of this PR

Reply:
ok, agree